src/router/index.ts

```diff
@@ -24,5 +25,10 @@ export default new Router({
       name: 'ManagePeople',
       component: ManagePeople,
     },
+    {
+      path: '/transactions/products',
+      name: 'ProductTransactions',
```

Comment:
is this user facing?

Reply:
what do you mean?

Reply:
is the name of a route user facing? i know in agritrader, the name part gets displayed in the header

Reply:
No I don't think so.